docs/technical-overview.md

```diff
@@ -0,0 +1,250 @@
+# Podium Request Portal — Technical Documentation
+
+## Project summary
+
+Podium Request Portal is an open source web-based application with which researchers can place requests for samples or data (including images) from Dutch biobanks and registries, and where these requests can be tracked and traced. The focus of this project is on supporting only essential (generic) steps of these request processes, keeping in mind that additional functionality might need to be added later on.
+
+Project source code is available in the following repositories:
+- https://github.com/thehyve/podium-registry — configuration registry and system health monitoring webapp;
+- https://github.com/thehyve/podium — core functionality.
+
+Project codebase is maintained by The Hyve.
+For technical questions you can contact alexey@thehyve.nl or gijs@thehyve.nl
```

Comment:
I would point to the issues page on Github: https://github.com/thehyve/podium/issues